src/server/vnparse.c

```diff
@@ -3566,6 +4117,14 @@ pbs_release_nodes_given_select(relnodes_input_t *r_input, relnodes_input_select_
 					} else if (
 						strcmp(pkvp[j].kv_keyw, "accelerator_memory") == 0) {
 						have->rl_accel_mem += to_kbsize(pkvp[j].kv_val);
+#if !(defined(PBS_MOM) || defined(PBS_PYTHON))
+					} else {
+						if ((rc = resc_limit_insert_other_res(have, pkvp[j].kv_keyw, pkvp[j].kv_val, TRUE))) {
```

Comment:
Could you please set rc outside of the if statement? Then check if (rc != 0)

Reply:
Addressed in commit https://github.com/PBSPro/pbspro/pull/1452/commits/8f309272ad00f7587c830cad07238cb40598e24a